openstef/tasks/create_basecase_forecast.py

```diff
@@ -42,6 +43,10 @@ def create_basecase_forecast_task(
         context: Contect object that holds a config manager and a database connection
 
     """
+    if pj.pipelines_to_run == PipelineType.TRAIN:
```

Comment:
Will this work? As pj.pipelines_to_run is a list now I think this will break.

Reply:
I think nicest is to add a pipelinetype for the basecase forecast and do a check similar to the ones in the train and forecasts pipelines

Reply:
Sure, I forgot to change it. Nice catch!